### ISSUE TITLE
[Logging] Allow include a set of initial labels for annotation

### ISSUE THREAD

**Issue by frascuchon** [recognai/rubrix#737]
Text classification and token classification task logging should include a set of labels/entities used for dataset annotation if we have no predictions/annotations at all

@dvsrepo @dcfidalgo

**Comment by dcfidalgo** [comment]
Isn't this partially covered by the UI's "Create new label" feature? Do you have a specific use case in mind?

**Comment by frascuchon** [comment]
it is quite tedious in cases where you have a large number of tags to include them manually. For example for an intent classifier

**Comment by dcfidalgo** [comment]
Maybe the UI could provide a text field where you can copy&paste a list of labels. 
Otherwise, do you propose a new parameter in the `rb.log` method, or rather in the `rb.*Record` classes? Or maybe introduce a "dataset" Datamodel?

**Comment by frascuchon** [comment]
My first idea is to include a `labels=[]` param in `rb.log`

```python
rb.log(records, name="test-dataset", tags=...., labels=["A","B","C"])
```
This should be a common param that are only processed for text classification and token classification. 

The use case should be interesting when data without annotations or predictions available, and should match better than including labels as mock predictions

**Comment by frascuchon** [comment]
Resolved in #1432
